include/ua_plugin_nodestore.h

```diff
@@ -83,6 +83,14 @@ typedef struct {
     /* Members specific to open62541 */         \
     void *context;
 
+/* Add counter only if subscriptions are enabled */
+#ifdef UA_ENABLE_SUBSCRIPTIONS
```

Comment:
I find it a bit more readable to ifdef-define a macro `UA_NODE_BASEATTRIBUTES_SUBSCRIPTIONS` first and then include that in `UA_NODE_BASEATTRIBUTES`.